cpp/ql/src/semmle/code/cpp/ir/implementation/aliased_ssa/internal/SSAConstruction.qll

```diff
@@ -153,10 +153,9 @@ private module Cached {
     )
   }
 
+  pragma[noopt]
   cached
-  Instruction getPhiOperandDefinition(
-    PhiInstruction instr, IRBlock newPredecessorBlock, Overlap overlap
-  ) {
+  Instruction getPhiOperandDefinition(Phi instr, IRBlock newPredecessorBlock, Overlap overlap) {
```

Comment:
Is there a particular reason to expose this as `Phi` (the IPA type) instead of `PhiInstruction`?

Reply:
OK, that seems like a good enough reason to keep it as the IPA type.